sqlglot/serde.py

```diff
@@ -0,0 +1,59 @@
+from sqlglot import expressions as exp
+
+
+def dump(node):
```

Comment:
add types

Reply:
Kind of a weird thing to annotate... but I gave it a shot